pkgs/by-name/sy/sysvol/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gtk4-layer-shell,
+  gtkmm4,
+  pkg-config,
+  nix-update-script,
+  wireplumber,
+  wrapGAppsHook4,
+}:
+
+stdenv.mkDerivation {
+  pname = "sysvol";
+  version = "0-unstable-2024-06-07";
+
+  src = fetchFromGitHub {
+    owner = "System64fumo";
+    repo = "sysvol";
+    rev = "56d7dcda4b246e71b2c6d29cbb2315bddf446032";
+    hash = "sha256-WOcy2R0El1vl57Zimb7Hoh9XYTnH/zJS1n+gcaTU4V8=";
+  };
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace-fail 'pkg-config' ''${PKG_CONFIG}
+  '';
+
+  nativeBuildInputs = [
+    pkg-config
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [
+    gtk4-layer-shell
+    gtkmm4
+    wireplumber
+  ];
+
+  makeFlags = [
+    "DESTDIR=${placeholder "out"}"
+  ];
+
+  passthru.updateScript = nix-update-script {
+    extraArgs = [ "--version" "branch" ];
+  };
+
+  meta = {
+    description = "A basic GTK4 volume indicator";
+    mainProgram = "sysvol";
+    homepage = "https://github.com/System64fumo/sysvol";
+    license = lib.licenses.wtfpl;
```

Comment:
I do not see any license file or mention in the repository, how did you figured out the project is under WTFPL License?

Reply:
the author told me to use WTFPL [here](https://github.com/System64fumo/sysvol/issues/1#issuecomment-2156330030).